components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
I'm happy to revert that commit and deal with catching any error in the `fixpyramid` subcommand. I don't know the original intention of the `proxy_to_instance` method and was only second guessing given its usage.